sci-biology/salmon/salmon-0.5.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2015 Gentoo Foundation
```

Comment:
`repoman` only tells me about the malformed CVS header (which I usually ignore because the git headers are different) - but I can fix this.